src/HPRT/toolkit/forms.py

```diff
@@ -103,3 +98,19 @@ class Meta:
             labels[index['id']] = str(i) + '. ' + index['body']
             i+=1
         json_data.close()
+
+class GeneralHealthForm(forms.ModelForm):
```

Comment:
For the new functions/classes we're adding, could you add some documentation on it while it's still fresh?  We'll also want to go back through and document what we've already written, but I think we should start better practices now.

There's some basic guidelines for documenting in Python - take a look at docstrings, and the basics for what to write in them